src/appleseed/renderer/modeling/object/curveobject.h

```diff
@@ -58,6 +58,25 @@ namespace renderer      { class ParamArray; }
 namespace renderer
 {
 
+enum class CurveBasis : unsigned char { Linear = 1, Bezier = 2, Bspline = 3, Catmullrom = 4 };
+
+const GScalar ar_bspline[16] = { 0.16666f,  0.66666f,  0.16666f,      0.0f,
```

Comment:
Follow naming conventions for constants. Move braces to own lines. Format like other such arrays are formatted (see colorspace.cpp for examples).